### PR TITLE
Image Embedder Script: use image locations from Table

### LINKED ISSUE #59: Improve scripting for ImageEmbedder
##### Image Analytics version

0.1.7

##### Orange version

3.5.dev

##### Expected behavior
One can call ImageEmbedder and pass Orange.data.Table as a parameter. ImEm then returns a data table with additional features.

##### Actual behavior
ImageEmbedder can be called only with a list of paths to images and it returns an array of arrays.

##### Additional info (worksheets, data, screenshots, ...)
Ideal workflow:

```
importer = ImportImages()
data, err = importer(<path-to-folder>)
embedder = ImageEmbedder()
embeddings = embedder(data)     # or embeddings = embedder(data, col="image")
```

### PR DESCRIPTION
**Pull request by jerneju**
##### Issue
Fixes #59.

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ajdapretnar** [comment]
1.) When embedder called without model parameter, one gets a TypeError: unhashable type: 'dict. Should default to 'inception-v3'.

2.) ImageEmbedder still doesn't return an enhanced Orange.data.Table, but a nd.array. I think there should be a nice way to get back a Table object.

3.) Please add a docstring with example.

**Comment by jerneju** [comment]
@ajdapretnar

File: orangecontrib/imageanalytics/image_embedder.py [hunk 1]
```diff
@@ -55,10 +57,18 @@ class ImageEmbedder(Http2Client):
     >>> image_file_paths = [...]
     >>> with ImageEmbedder(model='model_name', layer='penultimate') as embedder:
     ...    embeddings = embedder(image_file_paths)
+    or:
+    >>> from orangecontrib.imageanalytics.image_embedder import ImageEmbedder
+    >>> from orangecontrib.imageanalytics.import_images import ImportImages
+    >>> import_images = ImportImages()
+    >>> (images, err) = import_images("...")
```
Comment (ajdapretnar) [inline orangecontrib/imageanalytics/image_embedder.py]: no parentheses
`images, err = import_images("...")`

File: orangecontrib/imageanalytics/image_embedder.py [hunk 2]
```diff
@@ -55,10 +57,18 @@ class ImageEmbedder(Http2Client):
...
+    >>> image_embedder = ImageEmbedder()
+    >>> (embedded_images, skipped_images, num_skipped) = image_embedder(images)
```
Comment (ajdapretnar) [inline orangecontrib/imageanalytics/image_embedder.py]: same as above

**Comment by ajdapretnar** [comment]
The rest is good and works for me. 👍

**Comment by jerneju** [comment]
Thanks. Fixed.

**Pull request merged**